csunplugged/tests/topics/loaders/test_glossary_terms_loader.py

```diff
@@ -13,19 +14,39 @@ class GlossaryTermsLoaderTest(BaseTestWithDB):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.test_data = TopicsTestDataGenerator()
-        self.config_file = Mock()
         self.loader_name = "glossary_terms"
-        self.BASE_PATH = os.path.join(self.test_data.LOADER_ASSET_PATH, self.loader_name)
+        self.base_path = os.path.join(self.test_data.LOADER_ASSET_PATH, self.loader_name)
 
     def test_basic_config(self):
         folder = "glossary_folder"
 
-        glossary_loader = GlossaryTermsLoader(folder, self.config_file, self.BASE_PATH)
+        glossary_loader = GlossaryTermsLoader(base_path=self.base_path, content_path=folder)
+        glossary_loader.load()
+
+        glossary_objects = GlossaryTerm.objects.all()
+        self.assertEqual(1, len(glossary_objects))
+        term = glossary_objects[0]
+        self.assertSetEqual(set(["en"]), set(term.languages))
+        self.assertEqual("Glossary Term 1 English", term.term)
+        self.assertIn("English definition.", term.definition)
+
+    def test_translation(self):
+        folder = "glossary_translation"
+
+        glossary_loader = GlossaryTermsLoader(base_path=self.base_path, content_path=folder)
         glossary_loader.load()
 
         glossary_objects = GlossaryTerm.objects.all()
+        self.assertEqual(2, len(glossary_objects))
+
+        translated_term = GlossaryTerm.objects.get(slug="glossary-term-1")
+        untranslated_term = GlossaryTerm.objects.get(slug="glossary-term-2")
+
+        self.assertSetEqual(set(["en", "de"]), set(translated_term.languages))
+        self.assertSetEqual(set(["en"]), set(untranslated_term.languages))
 
-        self.assertQuerysetEqual(
-            glossary_objects,
-            ["<GlossaryTerm: Glossary Term 1>"]
-        )
```

Comment:
why are we removing this check that Queryset is as expected?

Reply:
See above.